core/components/organisms/verticalNav/__stories__/HoverableVerticalNav.story.jsx

```diff
@@ -306,18 +318,30 @@ const customCode = `() => {
   const [active, setActive] = React.useState({
     name: 'care_management.timeline'
   });
+  const [click, setClick] = React.useState(false);
+
+  const onToggleHandler = (expanded, type) => {
+    setExpanded(expanded);
+    if (type === 'click') {
+      if (expanded) {
+        setClick(!click);
+      } else {
+        setClick(!click);
+      }
+    }
+  };
 
   return (
     <div className="d-flex bg-secondary-lightest vh-100">
-      <Collapsible expanded={expanded} onToggle={setExpanded}>
-        <VerticalNav
-          menus={data}
-          active={active}
-          expanded={expanded}
-          onClick={setActive}
-        />
-      </Collapsible>
-      <div className="ml-6 d-flex flex-column">
+        <Collapsible expanded={expanded} onToggle={onToggleHandler}>
+          <VerticalNav
+            menus={data}
+            active={active}
+            expanded={expanded}
+            onClick={setActive}
+          />
+        </Collapsible>
+      <div className="mx-6 d-flex flex-column" style={{ width: click ? '70%' : '92%' }}>
```

Comment:
Please look for some other approach to set width instead of using style tag

Reply:
the other way was by making a class but then where to define the CSS for a story?